internal/cli/atlas/quickstart/quick_start.go

```diff
@@ -90,6 +92,18 @@ type Opts struct {
 	defaultValue        bool
 	Confirm             bool
 	store               store.AtlasClusterQuickStarter
+	defaultValues       DefaultOpts
+}
+
+type DefaultOpts struct {
+	ClusterName    string
+	Provider       string
+	Region         string
+	DBUsername     string
+	DBUserPassword string
+	IPAddresses    []string
+	SkipSampleData bool
+	SkipMongosh    bool // wont need
 }
```

Comment:
[nit] a bit of history, flags for non-interactive flows were an after though, so the way to know if it wasn't set was to keep them to default to blank and check if not set then ask

with this flip where we now set values and ask to confirm then we can do the correct thing, flags can have default values and as such they should reflect the default so when using `--help` you know what you get

All of this to say, this struct should not be needed, you could rely on the flags default option to achieve this with out adding a new struct

Reply:
oh, thanks for the background! that sounds good I didn't think of that for some reason let me try to change it - one question I have is what should we do with the `--default` flag then? looks like if we have no flags set we'll have same result as using `--default`

Reply:
We agreed that using default flags was not ideal and decided with another approach so I'll resolve.